src/delphi/vcl.castlecontrol.reg.pas

```diff
@@ -0,0 +1,45 @@
+{
+  Copyright 2022-2023 Michalis Kamburelis.
+
+  This file is part of "Castle Game Engine".
+
+  "Castle Game Engine" is free software; see the file COPYING.txt,
+  included in this distribution, for details about the copyright.
+
+  "Castle Game Engine" is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+  ----------------------------------------------------------------------------
+}
+
+{ Control with OpenGL context on a Delphi VCL form. }
```

Comment:
Please change this comment to 

```
{ Design-time functionality for the Castle Game Engine on Delphi VCL form. }
```

Reasons:

- The idea is to emphasize already here that this unit is only for design-time stuff.

- Also instead of "OpenGL context" we prefer more generic terms (in case of this unit "Castle Game Engine on Delphi VCL form" seems OK), as eventually we'll have other renderers.